test/functional/rpc_fundrawtransaction.py

```diff
@@ -928,6 +929,29 @@ def test_transaction_too_large(self):
         self.nodes[0].generate(10)
         assert_raises_rpc_error(-4, "Transaction too large", recipient.fundrawtransaction, rawtx)
 
+    def test_include_unsafe(self):
```

Comment:
As `FundTransaction()` has other callers like the `walletcreatefundedpsbt` and `send` RPCs, might be a good idea to have a similar `test_include_unsafe` tests that check for present/future regressions in `rpc_test_psbt.py` and `wallet_send.py`.

Reply:
Good idea, will do.

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/21359/commits/c3eb1994c24167bedb114e34b53dc7cbeb9ff403
I also added explicit support for these APIs, it makes more sense that way.